amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/common/parser/ParameterParsers.scala

```diff
@@ -501,6 +501,7 @@ case class Oas2ParameterParser(
       entry => {
         val req: Boolean = entry.value.as[Boolean]
         payload.annotations += RequiredParamPayload(req, entry.range)
```

Comment:
We were in fact parsing required as an annotation. Maybe we could remove this on our next major version. You can change the OAS emiiter to use the new field instead of the annotation

Reply:
Also update failing JSON-LDs